pkgs/by-name/ob/obj-magic/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "obj-magic";
+  version = "unstable-2020-09-20";
```

```suggestion
  version = "0.5-unstable-2020-09-20";
```
I found a version in the [source code](https://github.com/tapio/obj-magic/blob/f25c9b78cee6529a3295ed314d1c200677dc56c0/src/obj-magic.cpp#L19).

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md